index/05-chap5.Rmd

```diff
@@ -0,0 +1,139 @@
+                     Explanatory_Variables = c("Longitude & Latitude", "Longitude & Latitude & Elevation", "Solar Irradiance & best variable based on an hourly linear correlation computation", "Solar Irradiance & 2 bests variables based on an hourly linear correlation computation", "Solar Irradiance & 3 bests variables based on an hourly linear correlation computation", "2 bests variables based on linear correlation computation for every hour", "3 bests variables based on linear correlation computation for every hour", "4 bests variables based on linear correlation computation for every hour", "Variables with a linear correlation greater than 0.5", "Variables with a linear correlation greater than 0.3"))
+colnames(explvar) <- c("Statistical Method", "ID", "Explanatory variables")
+knitr::kable(explvar, caption = "Combination of explanatory variables used") %>%
+  kable_styling(latex_options = "scale_down") %>%
+  row_spec(0, bold = TRUE)
+```
+
+### Comparison of methods
+
+Once benchmark results are available, comparison of methods is possible. This comparison is based on the error of measures. In our case, RMSE and MAE were computed.
+
+MAE measures the average magnitude of the errors in a set of predictions, without considering their direction. It is the average over the test sample of the absolute differences between prediction and actual observation where all individual differences have equal weight.
+\[
+MAE = \frac{1}{n} \sum_{i=1}^{n}{ \lvert y_{j} - \widehat{y}_{j} \rvert}
+\]
+
+RMSE is a quadratic scoring rule that also measures the average magnitude of the error. It is the square root of the average of squared differences between prediction and actual observation.
+
+\[
+RMSE = \sqrt{\frac{1}{n} \sum_{i=1}^{n}{( y_{j} - \widehat{y}_{j} )^2}}
+\]
+
+They both express average model prediction error in units of the variable of interest. Both metrics can range from 0 to $\infty$ and are indifferent to the direction of errors. They are negatively-oriented scores, which means lower values are better.
+
+Since the errors are squared before they are averaged, the RMSE gives a relatively high weight to large errors. This means the RMSE is more useful because large errors are particularly undesirable in the project. (Chai, 2014)
+
+From the 10 methods compared, MAE and RMSE are computed and compared. The results are shown in the Figure \@ref(fig:meanerror). In this case, they both have the same behaviour. Multiple linear regression using coordinates to build models has a large error, the model is too simplified. Multiple linear regression using explanatory variables whose their linear correlations with temperature is greater than 0.3 has an error larger than the other methods too, this filter method is too flexible to return a valid model. A few methods have a similar error. In particular, that is the case when too many variables are chosen to build models.
```

Comment:
* In this case, they both have the same behaviour ==> it is not clear. What do you mean ?
* the model is too simplified ==> you can not use __the model__ as the RMSE and MAE are metrics resulting from a comparisons of various sets of model. The benchmark, for each hourly measurements dataset, compute a number of models equal to the number of stations (as much iterations as they are stations)